pkg/nsx/services/common/types.go

```diff
@@ -53,6 +53,8 @@ const (
 	AnnotationVPCNetworkConfig      string = "nsx.vmware.com/vpc_network_config"
 	AnnotationVPCName               string = "nsx.vmware.com/vpc_name"
 	DefaultNetworkConfigName        string = "default"
+	TagScopePodName                 string = "nsx-op/pod_name"
+	TagScopePodUID                  string = "nsx-op/pod_uid"
```

Comment:
Can we use "nsx-op/pod_uid" tag to distinguish pod and vm?

Reply:
Yes, I think we can.